ui/ProgressWindow.py

```
@@ -0,0 +1,658 @@
+                    self.messages_queue.put(event)
+
+                else:
+                    event = "%s [INFO]: A reboot is recommended" % (
+                        fn.datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S"),
+                    )
+                    self.messages_queue.put(event)
+
+                if os.path.exists("/usr/lib/modules/build"):
+                    shutil.rmtree("/usr/lib/modules/build", ignore_errors=True)
+
+                break
```

Ruff (0.7.0)

`break` inside `finally` blocks cause exceptions to be silenced

(B012)